packages/apputils/src/inputdialog.ts

```diff
@@ -312,6 +333,38 @@ class InputTextDialog extends InputDialogBase<string> {
   }
 }
 
+/**
+ * Widget body for input password dialog
+ */
+class InputPasswordDialog extends InputDialogBase<string> {
```

Comment:
Would it be simpler to just inherit `InputTextDialog` ?

Reply:
I don't think that it would change the length of the code. The boolean input dialog is also basically the same. All that differ is the constructor.